util/src/main/i18n/templates/match.properties

```diff
@@ -52,19 +52,56 @@ match.stats = Kills: {0}  |  Killstreak: {1}  |  Deaths: {2}  |  K/D: {3}
 # {1} = an amount of kills(number)
 match.stats.kills = Kills: {1} by {0}
 
+# {0} = an amount of kills(number)
+match.stats.kills.concise = Kills: {0}
+
 # {0} = a player name
 # {1} = a killstreak(number)
 match.stats.killstreak = Killstreak: {1} by {0}
 
+# {0} = a killstreak(number)
+match.stats.killstreak.concise = Highest killstreak: {0}
+
 # {0} = a player name
 # {1} = an amount of deaths(number)
 match.stats.deaths = Deaths: {1} by {0}
 
+# {0} = an amount of deaths(number)
+match.stats.deaths.concise = Deaths: {0}
+
+# {0} = numbers of kills divided by the number of deaths (kill-death ratio)
+match.stats.kd.concise = K/D: {0}
+
 # {0} = a player name
 # {1} = an amount of blocks
 match.stats.bowshot = Longest Shot: {1} blocks by {0}
 
-match.stats.overall = Match Stats
+# {0} = a player name
+# {1} = an amount of damage
+match.stats.damage = Damage: {1} by {0}
+
+# {0} = the number of damage done
+# {1} = the number of damage done with a bow
+# {2} = the number of damage taken
+match.stats.damage.concise = Damage given(bow) / taken: {0}({1}) / {2}
```

Comment:
This verbose stat implementation was made with more competitive players in mind, and this is something that can be useful to analyze performance in a comp match. This is also why I have decided to not pay attention to the stat setting while verbose stats is enabled

Damage given: All damage given by the player
(bow): All damage done by bow and arrow  by this player
taken: All damage taken by this player

I have yet to see how big numbers realistically get during a full match, since I only get to test on myself. But it is something that I hope to change if necessary after some testing

Reply:
For the first version of verbose stats, let's leave this out. I'm ok considering it later.

Reply:
Commit your changes and I'll review again